### PR TITLE
Import deleting Dataset.swift.

### PR DESCRIPTION
**Pull request by pschuh**

### CODE REVIEW

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please clean test tensorflow

**Comment by pschuh** [comment]
@swift-ci please clean test tensorflow

File: stdlib/public/TensorFlow/CMakeLists.txt [hunk 1]
```diff
@@ -60,6 +59,9 @@ if (TENSORFLOW_SWIFT_BINDINGS)
   file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES
     "${TENSORFLOW_SWIFT_BINDINGS}/*.swift")
   list(APPEND SOURCES "${TENSORFLOW_SWIFT_BINDINGS_SOURCES}")
+  file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES_CORE
```
Comment (dan-zheng) [inline stdlib/public/TensorFlow/CMakeLists.txt]: I believe you can use a single `file(GLOB_RECURSE ...)`:
```diff
 file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES
  "${TENSORFLOW_SWIFT_BINDINGS}/*.swift"
+ "${TENSORFLOW_SWIFT_BINDINGS}/Core/*.swift")
 list(APPEND SOURCES "${TENSORFLOW_SWIFT_BINDINGS_SOURCES}")
```

**Comment by dan-zheng** [comment]
@swift-ci Please clean test tensorflow macOS

File: stdlib/public/TensorFlow/CMakeLists.txt (hunk 1 above)
Comment (pschuh) [reply stdlib/public/TensorFlow/CMakeLists.txt]: SGTM. Will fix in the next PR.

**Comment by dan-zheng** [comment]
@swift-ci Please clean test tensorflow linux GPU

**Pull request merged**